readthedocs/api/v2/serializers.py

```diff
@@ -31,6 +31,12 @@ class Meta:
         )
 
 
+class AllEnvVarsField(serializers.ReadOnlyField):
+    def get_attribute(self, instance):
+        assert self.source_attrs == ['environment_variables']
+        return instance.environment_variables(public_only=False)
+
```

Comment:
Not sure if this is needed

Reply:
It is, that’s why painstakingly stepped through the code to add it :laughing: 

It’s `public_only=False` as opposed to the default `public_only=True`, which means that the V2 API keeps its behavior.

Reply:
I see now! Done.